thirdeye-ui/src/app/pages/home-page/home-page.component.tsx

```diff
@@ -168,6 +173,62 @@ export const HomePage: FunctionComponent = () => {
                                 />
                             </PageContentsCardV1>
                         </Grid>
+                        <Grid item sm={4} xs={12}>
+                            <PageContentsCardV1 fullHeight>
+                                <TotalSubscriptionGroupCount
+                                    classes={{
+                                        noDataIndicator: style.noDataIndicator,
+                                    }}
+                                    getSubscriptionGroupsStatus={
+                                        getSubscriptionGroupsStatus
+                                    }
+                                    subscriptionGroups={subscriptionGroups}
+                                />
+                            </PageContentsCardV1>
+                        </Grid>
+                    </Grid>
+                </Grid>
+
+                {!shouldHideDocumentation && (
+                    <Grid item xs={12}>
+                        <PageContentsCardV1>
+                            <RecommendedDocumentation
+                                onHideDocumentationClick={
+                                    handleHideDocumentationClick
+                                }
+                            />
+                        </PageContentsCardV1>
+                    </Grid>
+                )}
+
+                <Grid item sm={12} xs={12}>
+                    <Grid container justifyContent="space-between">
+                        <Grid item sm={8} xs={12}>
+                            <Typography variant="h5">
+                                {t("label.recent-entity", {
+                                    entity: t("label.anomalies"),
+                                })}
+                            </Typography>
+                        </Grid>
+
+                        <Grid item sm={4} xs={12}>
+                            <AnomalyRangeDropdown
+                                anomalyStartTime={anomalyStartTime}
+                                setAnomalyStartTime={setAnomalyStartTime}
+                            />
+                        </Grid>
+                    </Grid>
```

Comment:
Let's vertical align the header text since the dropdown seems pretty big with `alignItems="center"` This would require removing the preferred sizing configuration. So something like:

```
 <Grid container alignItems="center" justifyContent="space-between">
                        <Grid item>
                            <Typography variant="h5">
                                {t("label.recent-entity", {
                                    entity: t("label.anomalies"),
                                })}
                            </Typography>
                        </Grid>

                        <Grid item>
                            <AnomalyRangeDropdown
                                anomalyStartTime={anomalyStartTime}
                                setAnomalyStartTime={setAnomalyStartTime}
                            />
                        </Grid>
                    </Grid>
```

Reply:
Implemented this fix. Will this do, or should I make more changes?

![Screenshot_150592397](https://user-images.githubusercontent.com/20799363/206148037-ffc10cd4-41ef-4f85-937a-566a6a246140.png)

![Screenshot_150592398](https://user-images.githubusercontent.com/20799363/206148126-14cf925b-4454-45be-a7c7-b129fd7b29d7.png)